src/lib/fs-db.js

```diff
@@ -108,54 +121,54 @@ class FsDb {
   unloadDir(dirname) {
     const relpath = this.toRelpath(dirname);
     const map = this.mapFor(dirname);
-    const dir = path.basename(dirname);
+    const dir = path.posix.basename(dirname);
     map.delete(dir);
     this.logger.trace({ relpath }, 'unloaded directory');
-    if (this.unloadable(map)) { this.unloadDir(path.dirname(dirname)); }
+    if (this.unloadable(map)) { this.unloadDir(path.posix.dirname(dirname)); }
   }
 
   unloadFile(filepath, emit = true) {
     const relpath = this.toRelpath(filepath);
-    const file = path.basename(filepath);
+    const file = path.posix.basename(filepath);
     const originalExisted = typeof this.read(relpath) !== 'undefined';
     if (originalExisted) {
       const map = this.mapFor(filepath);
       map.delete(file);
       this.logger.trace({ relpath }, 'unloaded file');
-      if (this.unloadable(map)) { this.unloadDir(path.dirname(filepath)); }
+      if (this.unloadable(map)) { this.unloadDir(path.posix.dirname(filepath)); }
       if (emit) { this.emit('removed', relpath); }
     }
   }
 
   onFileAdded(filepath) {
-    const relpath = this.toRelpath(filepath);
+    const relpath = this.toRelpath(filepath.replace(/[\\]/g, '/'));
     this.logger.trace({ relpath }, 'added to filesystem');
     this.loadFile(filepath);
   }
 
   onFileChanged(filepath) {
     const relpath = this.toRelpath(filepath);
     this.logger.trace({ relpath }, 'changed in filesystem');
-    this.loadFile(filepath);
+    this.loadFile(filepath.replace(/[\\]/g, '/'));
```

Comment:
Instead of using the replace here, I think you could use on setupWatcher like this
`.on('change', path => this.onFileChanged(path.replace(/[\\]/g, '/')) )`
but I think is almost the same and not sure if it will work, I'm not testing on windows.